cli/js/mkdir_test.ts

```diff
@@ -15,11 +15,12 @@ unitTest(
   { perms: { read: true, write: true } },
   function mkdirSyncMode(): void {
     const path = Deno.makeTempDirSync() + "/dir";
-    Deno.mkdirSync(path, { mode: 0o755 }); // no perm for x
+    Deno.mkdirSync(path, { mode: 0o737 });
     const pathInfo = Deno.statSync(path);
     if (pathInfo.mode !== null) {
       // Skip windows
-      assertEquals(pathInfo.mode & 0o777, 0o755);
+      // assertEquals(pathInfo.mode, 0o737 & ~Deno.umask());
+      assertEquals(pathInfo.mode & 0o777, 0o715); // assume umask 0o022
```

Comment:
Please explicitly test behavior in windows using `if (Deno.build.os == "win")` ?

Reply:
Ok, the `mode !== null` check was there already; I'll be glad to change it to a more explicit check. I expect if the whole test only makes sense on Unix, I should have a `skip: Deno.build.os === "win"` option on the unitTest?

Reply:
I guess since windows shouldn't fail if mkdirSync is provided with a mode, it would be good to test that, but the assert checking the mode should only be done done on non-windows.

Reply:
Two things:

1. So you're expressing a vote here (also [here](https://github.com/denoland/deno/pull/4287#discussion_r390022278)) that ops like mkdir/mkdirSync with an explicit mode supplied shouldn't throw on Windows? See #4306, @kitsonk preferred instead that it would throw. Either way, I'll check against `Deno.build.os` to disable the relevant part of the test.

2. Based on [another of your comments](https://github.com/denoland/deno/pull/4287#discussion_r390022552), I'll make all of the current batch of PRs use the explicit umask-querying. Note that will make their tests start failing until the umask PR is merged into master. (And then these PRs are synched.)

Reply:
This PR shouldn't be merged until those two issues are settled.